### PR TITLE
Add more button codes

### PR DESCRIPTION
**Pull request by presslab-us**
This adds some more button codes. I only needed the NAVIGATION code but I figure why not add all the ones I found.

### CODE REVIEW

File: aasdk_proto/ButtonCodeEnum.proto [hunk 1]
```diff
@@ -42,6 +42,14 @@ message ButtonCode
         PREV = 0x58;
         PLAY = 0x7E;
         PAUSE = 0x7F;
+        MUSIC = 0xD1;
         SCROLL_WHEEL = 65536;
+        MEDIA = 65537;
```
Comment (htruong) [inline aasdk_proto/ButtonCodeEnum.proto]: Is there a reason why you have decimal values instead of hex values like the rest of them in this file?

Comment (presslab-us) [reply aasdk_proto/ButtonCodeEnum.proto]: I just copied the constants like they were in the other project.

Comment (htruong) [reply aasdk_proto/ButtonCodeEnum.proto]: I think perhaps, the better way to do this is to fix the one line (`SCROLL_WHEEL`) where it is inconsistent and add everything as hex.

**Comment by presslab-us** [comment]
Is anything else needed to get this merged?

**Comment by htruong** [comment]
Thank you for the modification, request merged.

**Pull request merged**